packages/terra-form-input/src/Input.jsx

```diff
@@ -40,6 +40,10 @@ const propTypes = {
    * Content to be displayed as the name.
    */
   name: PropTypes.string,
+  /**
+   * Pattern to match when input type is used.
```

Comment:
This comment is a little confusing with the is used ending. I would just say Pattern for the input value.

Reply:
Comment changed to suggested text.